crates/types/src/traits/auction_results_provider.rs

```diff
@@ -23,9 +24,31 @@ pub trait HasUrls {
 pub trait AuctionResultsProvider<TYPES: NodeType>: Send + Sync + Clone {
     /// The AuctionSolverResult is a type that holds the data associated with a particular solver
     /// run, for a particular view.
-    type AuctionResult: HasUrls;
+    type AuctionResult: HasUrls + Send;
 
     /// Fetches the auction result for a view. Does not cache the result,
     /// subsequent calls will invoke additional wasted calls.
     async fn fetch_auction_result(&self, view_number: TYPES::Time) -> Result<Self::AuctionResult>;
```

Comment:
In both `fetch_auction_result` and `fetch_bundles` we'll want a timeout.  If we get the results we are looking for, we return immediately.  Otherwise, we return whatever results we get by the timeout (for both the auction results and bundles) .  I think that timeout logic still needs to be implemented here.

Reply:
Added timeouts! Added constants for the durations, and the `fetch_bundles` method now has timeouts on both `fetch_auction_result` and the GET request to each builder!